angular-spa-ui/js/common/main-ctrl.js

```diff
@@ -0,0 +1,152 @@
+"use strict";
+
+module.exports = function ($scope, promises, getTemplate, configService, appStorage, appLocalStorage) {
+
+    var config = configService.getConfig('appConfig'),
+        paths = config.paths,
+        userData = appLocalStorage.getItem('userData') || {};
+    paths.userPath += GLOBAL_USER_ID;
+
+    function initUserData(data) {
+        userData['userId'] = data.data._id;
+        userData['userName'] = data.data.name;
+        userData['userEmail'] = data.data.email;
+        userData['published'] = data.data.published;
```

Comment:
If you storing 'published' flag you should use it. user.published==false means that the user id banned or blocked or has no permissions to do anything